clickhouse_plantuml/__main__.py

```diff
@@ -78,6 +78,12 @@ def parse_args() -> Namespace:
         help="tables whitelist to describe. If set, only mentioned tables will"
         "be queried from the server",
     )
+    clickhouse.add_argument(
+        "--no-merge-mat-views",
```

Comment:
Let's keep it consistent, please

```suggestion
        "--no-merge-matviews",
```